redesign-project/src/subscribe.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: layouts/default.njk
+title: Subscribe
+---
+
+<h1 class="subscribe_header">
+    {#
+    <meta charset="utf-8"> #}
+    Subscribe to our newsletter!
+</h1>
+
+<body>
+    <form action="/my-handling-form-page" method="post" class="form">
+        <div>
+            <label for="name">First Name:</label> <input type="text" id="firstname" name="user_name">
```

Comment:
The `for` attribute on a `label` must match the `id` on the input it relates to:
```suggestion
             <label for="firstname">First Name:</label> <input type="text" id="firstname" name="user_name">
```